restli-tools/src/main/resources/apiVmTemplates/action.vm

```diff
@@ -73,22 +63,12 @@ Copyright (c) 2021 LinkedIn Corp.
       #if(${method.isEntityAction()})
       $spec.keyClassName $spec.idName,
```

Comment:
same here, for action without any params

Reply:
same here, check on line 60 gurads this.